src/Scriban/Runtime/StandardMemberRenamer.cs

```diff
@@ -12,19 +11,18 @@ namespace Scriban.Runtime
     /// </summary>
     public sealed class StandardMemberRenamer
     {
-        public static readonly MemberRenamerDelegate Default = Rename;
+        public static readonly MemberRenamerDelegate Default = member => Rename(member.Name);
 
         /// <summary>
-        /// Renames a camel/pascalcase member to a lowercase and `_` name. e.g `ThisIsAnExample` becomes `this_is_an_example`.
+        /// Renames a camel/pascalcase method name to a lowercase and `_` name. e.g `ThisIsAnExample` becomes `this_is_an_example`.
         /// </summary>
-        /// <param name="member">The member to rename</param>
+        /// <param name="name">The method name to rename</param>
         /// <returns>The member name renamed</returns>
-        public static string Rename(MemberInfo member)
+        public static string Rename(string name)
```

Comment:
Could you keep the previous implementation (call (string) from (memberInfo) ) so that it is not a breaking change

Reply:
What if I create a new method called `DoRename` that accepts a string?

Reply:
As, it's just an overload of the function, it should keep the same name.

Reply:
Even better.